lib/routes/data/styles.js

```diff
@@ -63,6 +63,20 @@ module.exports = {
       ],
     ],
   },
+  contentData: {
+    name: 'Careers',
+    description: 'A section realted to joining Watson and what the culture of Watson is about',
```

Comment:
Hmm... I pulled that description directly from the Careers content type. But for clarity, I can update here.